packages/frontend/src/components/VisualizationConfigs/ChartConfigPanel/AxesOptions.tsx

```diff
@@ -0,0 +1,258 @@
+import {
+    Field,
+    getAxisName,
+    getItemId,
+    getItemLabelWithoutTableName,
+    isNumericItem,
+    TableCalculation,
+} from '@lightdash/common';
+import {
+    Checkbox,
+    Group,
+    SegmentedControl,
+    Stack,
+    Switch,
+    Text,
+    TextInput,
+} from '@mantine/core';
+import { FC, useCallback } from 'react';
+import { useToggle } from 'react-use';
+import { useTracking } from '../../../providers/TrackingProvider';
+import { EventName } from '../../../types/Events';
+import { useVisualizationContext } from '../../LightdashVisualization/VisualizationProvider';
+
+interface MinMaxProps {
+    label: string;
+    min: string | undefined;
+    max: string | undefined;
+    setMin: (value: string | undefined) => void;
+    setMax: (value: string | undefined) => void;
+}
+
+const AxisMinMax: FC<MinMaxProps> = ({ label, min, max, setMin, setMax }) => {
+    const [isAuto, toggleAuto] = useToggle(!(min || max));
+    const { track } = useTracking();
+
+    const clearRange = useCallback(() => {
+        if (!isAuto) {
+            setMin(undefined);
+            setMax(undefined);
+        }
+        return;
+    }, [isAuto, setMin, setMax]);
+
+    return (
+        <>
+            <Switch
+                label={label}
+                checked={isAuto}
+                size="sm"
```

Comment:
I think this could be `xs` perhaps? 

<img width="555" alt="image" src="https://github.com/lightdash/lightdash/assets/7611706/b04b41bd-9641-434f-ab3a-6b4118234853">

Reply:
Same as above, sure!